services/frontend/.env.production

```diff
@@ -0,0 +1,8 @@
+REACT_APP_GRAPHQL_HTTP_URL=https://graphql-jungle.eosrate.io/v1alpha1/graphql
+REACT_APP_API_URL=https://jungle.eosio.cr
```

Comment:
@JustinCast @kuronosec 

are we using jungle or mainnet in production?

Reply:
@xavier506  I updated the file. Please take another look